datacube/drivers/postgres/_core.py

```diff
@@ -157,7 +157,15 @@ def update_schema(engine: Engine):
     # Empty, as no schema changes have been made recently.
     # -> If you need to write one, look at the Git history of this
     #    function for some examples.
-    return
+    
+    # Post 1.8 DB Federation triggers
+    from datacube.drivers.postgres._triggers import install_timestamp_trigger
+    _LOG.info("Adding Update Triggers")
```

Comment:
This is going to log every time they run `init`, even if they already have applied this update.

By convention, I usually wrapped them in a check previously:

```python
if pg_column_exists(engine, schema_qualified('dataset'), 'updated'):
    _LOG.info("Adding Update Triggers")
    with engine.connect() as c:
        c.execute('begin')
        install_timestamp_trigger(c)
        .... etc
```
(of course, no need to check other columns exist as they're all applied together in one transaction.)

The actual SQL looks idempotent so this is all an extremely minor gripe.